api/rest/client/transports.go

```diff
@@ -41,11 +40,15 @@ func (c *defaultClient) defaultTransport() {
 func (c *defaultClient) enableLibp2p() error {
 	c.defaultTransport()
 
-	pid, addr, err := api.Libp2pMultiaddrSplit(c.config.APIAddr)
+	pinfo, err := peerstore.InfoFromP2pAddr(c.config.APIAddr)
```

Comment:
A function in `peerstore` does mostly the same as a function we had, so started using that one.